src/app/shell/details/side-menu/contacts/contacts.component.html

```diff
@@ -30,11 +30,11 @@ <h4 class="card-title">{{ 'CONTACTS' | translate }}</h4>
   </p>
 
   <div class="social" fxLayout="row" fxLayoutAlign="space-between center">
-    <a href="{{ contactsData.instagram }}" [ngClass]="contactsData.instagram ? 'link' : 'disable'" target="_blank"
+    <a href="{{ contactsData.instagram }}" *ngIf="contactsData.instagram" [ngClass]="contactsData.instagram ? 'link' : 'disable'" target="_blank"
```

Comment:
it is better to put the *ngIf directive at the beginning. 
and it seems that now we don't need ngClass, since disabled button will be always hidden